supabase/functions/_backend/public/build/request.ts

```
@@ -0,0 +1,206 @@
+        },
+        body: JSON.stringify({
+          userId: org_id, // Use org_id as anonymized identifier
+          fastlane: {
+            lane: platform,
+          },
+          credentials: credentials || {},
+        }),
+      })
+
+      if (builderResponse.ok)
+        builderJob = await builderResponse.json() as BuilderJobResponse
```

ESLint

[error] Expect curly brackets

(antfu/curly)